ContractsApi/V1/Boundary/Requests/CreateContractRequestObject.cs

```diff
@@ -21,6 +21,7 @@ public class CreateContractRequestObject
         public string CostCentre { get; set; }
         public string Brma { get; set; }
         public bool? IsActive { get; set; }
+        public bool? IsApproved { get; set; } = false;
```

Comment:
That's a very good question which will open your eyes to an intriguing (?) aspect of the BTA FE logic.
The CreateContract payload is replicated throughout the FE, as changing any field of the Contract section will need to create a contract if there isn't one already. All those fields have separate editors, which will each either create a new contract or update an existing one via the API.
By leaving this field as nullable with a default value, we can avoid having to add this parameter to some 15 files on the FE.

Reply:
Hmm... yeah that sounds like a lot of work :thinking:

I could be remembering this wrong, but I think when you don't provide a payload field in .NET the model binder set's it to the default value for that type. So if it was a hard non-nullable boolean, it was not in the payload, it would default to `false`, just like when an integer field is not provided, it defaults to "0".

But again, it's a cosmetic thing, so we can leave it as is.

Reply:
Ah! that settles it then. I don't think it's just a cosmetic thing, as having a nullable bool field with a default to `false` is essentially more code to reach the same result as a non nullable.